go.mod

```diff
@@ -27,6 +27,7 @@ require (
 	github.com/prometheus/client_golang v1.4.0
 	github.com/prometheus/client_model v0.2.0
 	github.com/prometheus/common v0.9.1
+	github.com/rancher/machine v0.13.0 // indirect
```

Comment:
how did this get rancher/machine indirectly? that's kind of odd...

Reply:
Not exactly sure, I'll look into this. `go.mod` and `go.sum` shouldn't change